src/tests/markdown.spec.js

```diff
@@ -120,9 +105,11 @@ describe('Markdown though editor', () => {
 		expect(markdownThroughEditor('[bar\\\\]: /uri\n\n[bar\\\\]')).toBe('[bar\\\\](/uri)')
 	})
 	test('images', () => {
+		// Inline images
 		expect(markdownThroughEditor('text ![test](foo) moretext')).toBe('text ![test](foo) moretext')
-		// regression introduced in #3282. To be fixed in #3428.
-		expect(markdownThroughEditor('![test](foo)')).toBe('![test](foo)\n\n')
+		// regression introduced in #3282. See issue #3428.
```

Comment:
This comment is outdated now :smiley: - we don't need it anymore.
```suggestion
```

Reply:
I let the comment stay there to have some information why a test was added, but I can remove it as well :)

Reply:
Okay... cool. I meant the comment as documenting a regression - now it's documenting a fixed regression - i guess that's fine. Either way it points to a relevant issue.